lib/buildkite/test_collector/ci.rb

```diff
@@ -82,4 +83,16 @@ def circleci
       "number" => ENV["CIRCLE_BUILD_NUM"],
     }
   end
+
+  def codeship
+    {
+      "CI" => "codeship",
+      "key" => "#{ENV["CI_BUILD_ID"]}",
```

Comment:
Is the ID globally unique or only unique to the project?

Do we want to incorporate the `CI_PROJECT_ID`?

Reply:
I had a peek at their API, it seems projects and builds are globally unique with UUID-shaped identifiers. 👍️